Add Trino Fest 2024 recap (#673)
trinodb/trino.io: _posts/2024-06-18-trino-fest-recap.md

---
layout: post
title: "Trino Fest 2024 recap"
author: "Manfred Moser, Cole Bowden, Monica Miller"
excerpt_separator: <!--more-->
image: /assets/blog/trino-fest-2024/recap-banner.png
---

tbd

> <!--more-->

Reviewer: Trino Fest 2024 is successfully in the books! While over 100 enthusiastic
members of the community gathered in Boston, over 650 virtual attendees 
joined us worldwide to learn from our expert speakers as they discussed
topics such as table formats, enhancements and optimizations, and use
cases with Trino both large and small.

> <!--more-->

Reviewer: ```suggestion
Trino Fest 2024 is successfully in the books! While over 100 enthusiastic
members of the community gathered in Boston, over 650 virtual attendees
joined us worldwide to learn from our expert speakers as they discussed
topics such as table formats, enhancements and optimizations, and use
cases with Trino both large and small.
<!--more-->
```

## Re
tbd

## Sessions

> * [What’s new in Trino this summer](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [What’s new in Trino this summer](https://www.youtube.com/watch?v=mk3n0_tAdZY){:target="_blank"}
```

  presented by Martin Traverso from
  [Starburst](https://www.starburst.io){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Reducing query cost and query runtimes of Trino powered analytics platforms](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [Reducing query cost and query runtimes of Trino powered analytics platforms](https://youtu.be/mk3n0_tAdZY?si=8d1oyZmpxL2Luj_5){:target="_blank"}
```

> * [Reducing query cost and query runtimes of Trino powered analytics platforms](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [Reducing query cost and query runtimes of Trino powered analytics platforms](https://youtu.be/11yhPXIXiBY?si=O4x4ac2X8Kq0QPmA){:target="_blank"}
```

  presented by Jonas Irgens Kylling from
  [Dune](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Enhancing Trino's query performance and data management with Hudi: innovations and future](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [Enhancing Trino's query performance and data management with Hudi: innovations and future](https://youtu.be/JMzS2BbeK0E?si=wkJxUGt6jsG6-xz2){:target="_blank"}
```

  presented by Ethan Guo from
  [Onehouse](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Trino Engineering @ Microsoft](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [Trino Engineering @ Microsoft](https://www.youtube.com/watch?v=t7ndqYUhKSA){:target="_blank"}
```

  presented by George Fisher and Ishan Patwa from
  [Microsoft](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Enhancing data governance in Trino with the OpenLineage integration](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: [](url)
```suggestion
* [Enhancing data governance in Trino with the OpenLineage integration](https://www.youtube.com/watch?v=A7hj1M7IYj8){:target="_blank"}
```

  presented by Alok Kumar Prusty from
  [Apple](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Best practices and insights when migrating to Apache Iceberg for data engineers](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [Best practices and insights when migrating to Apache Iceberg for data engineers](https://www.youtube.com/watch?v=dKQ2zShNlyQ){:target="_blank"}
```

  presented by Amit Gilad from
  [Cloudinary](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Trino query intelligence: insights, recommendations, and predictions](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [Trino query intelligence: insights, recommendations, and predictions](https://www.youtube.com/watch?v=K3iSXOJNaSQ){:target="_blank"}
```

  presented by Marton Bod from
  [Apple](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [The open source journey of the Trino Delta Lake Connector](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [The open source journey of the Trino Delta Lake Connector](https://www.youtube.com/watch?v=mPfRYdvDcMo){:target="_blank"}
```

  presented by Marius Grama from
  [Starburst](https://www.starburst.io){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Tiny Trino; new perspectives in small data](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [Tiny Trino; new perspectives in small data](https://www.youtube.com/watch?v=ZcY9LJDdB6Y){:target="_blank"}
```

  presented by Ben Jeter and Thomas Zugibe from
  [Executive Homes](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Bridging the divide: running Trino SQL on a vector data lake powered by Lance](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [Bridging the divide: running Trino SQL on a vector data lake powered by Lance](https://www.youtube.com/watch?v=jmOsVbGfon0){:target="_blank"}
```

  presented by Lei Xu from LanceDB and Noah Shpak from Character.ai
  [](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [How FourKites runs a scalable and cost-effective log analytics solution to handle petabytes of logs](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [How FourKites runs a scalable and cost-effective log analytics solution to handle petabytes of logs](https://www.youtube.com/watch?v=xdCZBQJt-0g){:target="_blank"}
```

  presented by Arpit Garg from
  [FourKites](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Observing Trino](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [Observing Trino](https://www.youtube.com/watch?v=v7p72Ggcc5I){:target="_blank"}
```

  presented by Matt Stephenson from
  [Starburst](https://www.starburst.io){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Accelerate Performance at Scale: Best Practices for Trino with Amazon S3](https://www.youtube.com/watch?v=){:target="_blank"}

Reviewer: ```suggestion
* [Accelerate Performance at Scale: Best Practices for Trino with Amazon S3](https://www.youtube.com/watch?v=cjUUcHlUKxQ){:target="_blank"}
```

---
layout: post
title: "Trino Fest 2024 recap"
author: "Manfred Moser, Cole Bowden, Monica Miller"
excerpt_separator: <!--more-->
image: /assets/blog/trino-fest-2024/recap-banner.png
---

tbd
Trino Fest 2024 is successfully in the books! While over 100 enthusiastic
members of the community gathered in Boston, over 650 virtual attendees
joined us worldwide to learn from our expert speakers as they discussed
topics such as table formats, enhancements and optimizations, and use
cases with Trino both large and small.
<!--more-->

## Re
tbd

## Sessions

> * [What’s new in Trino this summer](https://www.youtube.com/watch?v=mk3n0_tAdZY){:target="_blank"}

Reviewer 2: Martin recapped everything that's happened in Trino over the last six months, taking a look at the biggest new features and how Trino development is going better than ever. He also gave a sneak peek at what we can expect soon in Trino.

  presented by Martin Traverso from
  [Starburst](https://www.starburst.io){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Reducing query cost and query runtimes of Trino powered analytics platforms](https://www.youtube.com/watch?v=11yhPXIXiBY){:target="_blank"}

Reviewer 2: Jonas gave a detailed talk about how Dune has improved their performance of Trino with a few key tweaks. That includes leveraging caching with Alluxio, advanced cluster management, and storing, sampling, and filtering query results.

  presented by Jonas Irgens Kylling from
  [Dune](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Enhancing Trino's query performance and data management with Hudi: innovations and future](https://www.youtube.com/watch?v=JMzS2BbeK0E){:target="_blank"}

Reviewer 2: Ethan gave a look into development on Hudi and Trino's Hudi connector, explaining multi-modal indexing and how it can improve query performance. He also gave an overview of the roadmap and future of the connector.

  presented by Ethan Guo from
  [Onehouse](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Trino Engineering @ Microsoft](https://www.youtube.com/watch?v=t7ndqYUhKSA){:target="_blank"}

Reviewer 2: George and Ishan gave a deep dive into what's been going on with Microsoft's deployment and management of Trino. This included clients and integrations, result caching, a sharded SQL connector, deep debugging and monitoring, and seamless security integration with Azure.

  presented by George Fisher and Ishan Patwa from
  [Microsoft](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Enhancing data governance in Trino with the OpenLineage integration](https://www.youtube.com/watch?v=A7hj1M7IYj8){:target="_blank"}

Reviewer 2: Alok's lightning talk was all about how Apple deployed OpenLineage, an open framework for data lineage collection and analysis, and built a Trino plugin to publish OpenLineage complaint events that can be viewed and monitored.

  presented by Alok Kumar Prusty from
  [Apple](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Best practices and insights when migrating to Apache Iceberg for data engineers](https://www.youtube.com/watch?v=dKQ2zShNlyQ){:target="_blank"}

Reviewer 2: Amit shared how Cloudinary expanded their data lake to use Apache Iceberg. He demonstrated how moving from Snowflake to an open table format allowed them to reduce storage costs and leverage different query and processing engines to run more powerful analytics at scale.

  presented by Amit Gilad from
  [Cloudinary](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Trino query intelligence: insights, recommendations, and predictions](https://www.youtube.com/watch?v=K3iSXOJNaSQ){:target="_blank"}

Reviewer 2: Marton's lightning talk explored how Apple has monitored and stored metadata for every Trino query execution, then used that data for for real-time cluster dashboarding, self-service troubleshooting, and automatic generation of recommendations for users.

  presented by Marton Bod from
  [Apple](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [The open source journey of the Trino Delta Lake Connector](https://www.youtube.com/watch?v=mPfRYdvDcMo){:target="_blank"}

Reviewer 2: Marius went into a deep dive on all the work and collaboration that's gone into making the Delta Lake connector in Trino a robust, first-class connector. Casual discussions, engineers working together, GitHub issues filed by the community, and innovative contributions have all come together, and Marius' talk shows why an open source community is so powerful.

  presented by Marius Grama from
  [Starburst](https://www.starburst.io){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Tiny Trino; new perspectives in small data](https://www.youtube.com/watch?v=ZcY9LJDdB6Y){:target="_blank"}

Reviewer 2: Ben and Tommy explore how Executive Homes uses Trino's robust suite of integrations to handle data at a small scale. Instead of petabytes, how about a handful of gigabytes in several different systems? It's something that Trino is well-equipped to handle thanks to how well-supported it is in the data ecosystem, and they explain why.

  presented by Ben Jeter and Thomas Zugibe from
  [Executive Homes](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Bridging the divide: running Trino SQL on a vector data lake powered by Lance](https://www.youtube.com/watch?v=jmOsVbGfon0){:target="_blank"}

Reviewer 2: Lei and Noah give an overview of LanceDB, how it works, and what makes it a great database for multimodal AI. Then they dive into a Trino connector for Lance, and explore how Trino slots into Character.AI's workload to blend analytics with training and generating new models.

  presented by Lei Xu from LanceDB and Noah Shpak from Character.ai
  [](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [How FourKites runs a scalable and cost-effective log analytics solution to handle petabytes of logs](https://www.youtube.com/watch?v=xdCZBQJt-0g){:target="_blank"}

Reviewer 2: With nearly a petabyte of logs being managed at FourKites, it shouldn't be a huge surprise that they've turned to Trino to handle understanding and analyzing them. Arpit discusses how they've scaled log ingestion, strategically used S3 with Parquet to minimize storage costs, transformed and extracted those logs at scale, and leveraged Trino to search and explore the datasets with Superset as a frontend for visualization.

  presented by Arpit Garg from
  [FourKites](){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Observing Trino](https://www.youtube.com/watch?v=v7p72Ggcc5I){:target="_blank"}

Reviewer 2: Starburst has built a comprehensive observability platform around Trino to better serve its users and customers. Matt explored all the components of it, including how to integrate with Jaeger, Prometheus, and ELK.

  presented by Matt Stephenson from
  [Starburst](https://www.starburst.io){:target="_blank"}.
  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)

> * [Accelerate Performance at Scale: Best Practices for Trino with Amazon S3](https://www.youtube.com/watch?v=cjUUcHlUKxQ){:target="_blank"}

Reviewer 2: Dai's talk explores best practices to get the most out of using Trino in conjunction with Amazon S3. He discusses partitioning, scaling workloads, reducing latency, and resolving common bottlenecks, providing valuable insights for anyone trying to manage and deploy Trino with S3.
